simplq/src/components/common/Popup/TempQueueWarning.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import IconButton from '@material-ui/core/IconButton';
+import Collapse from '@material-ui/core/Collapse';
+import CloseIcon from '@material-ui/icons/Close';
+import Grid from '@material-ui/core/Grid';
+import { isLoggedIn } from '../../../services/auth';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    width: '100%',
+    background: '#FF8000',
+    color: '#FFFFFF',
+    top: '0',
+    position: 'absolute',
+    zIndex: '100',
+  },
+}));
+
```

Comment:
@shubh128, could you move this styling code to an scss file and import it for use. That's how it's done everywhere else, so let's keep things consistent.

Reply:
If possible, try to reduce the usage of material UI. This is not a hard requirement, we just don't want to be too tightly coupled with any styling library. If this is too hard to do, leave it for now.

Reply:
Let's keep it as a hard requirement here. Except for the `CloseIcon`, can you avoid all other material ui usage?